src/main/java/frc/robot/testcontainers/ArmContainer.java

```diff
@@ -2,48 +2,89 @@
 
 import edu.wpi.first.wpilibj2.command.button.CommandXboxController;
 import edu.wpi.first.wpilibj2.command.Command;
+import edu.wpi.first.wpilibj2.command.Commands;
+
+import edu.wpi.first.wpilibj2.command.SequentialCommandGroup;
 import edu.wpi.first.wpilibj2.command.button.Trigger;
 
 import frc.robot.BaseContainer;
 import frc.robot.Constants.OperatorConstants;
+import frc.robot.Constants;
 import frc.robot.subsystems.ArmSubsystem;
+import frc.robot.subsystems.IntakeSubsystem;
 import frc.robot.commands.Arm.SpinNoteContainerMotor;
 import frc.robot.commands.Arm.StopNoteContainerMotor;
-import frc.robot.commands.Arm.SwingBack;
-import frc.robot.commands.Arm.SwingForward;
-import frc.robot.commands.Arm.SwingServo;
+
+import frc.robot.commands.Arm.SwingForwardServo;
+import frc.robot.commands.Arm.SwingBackServo;
+import frc.robot.commands.Arm.SpinAndSwing;
+import frc.robot.commands.Arm.SpinArmPID;
+import frc.robot.commands.Arm.SpinToArmAngle;
+
+import frc.robot.commands.IntakeCommands.IntakePassNoteToPizzaBox;
+import frc.robot.commands.IntakeCommands.IntakePickUpFromGround;
+
 
 import edu.wpi.first.wpilibj.shuffleboard.Shuffleboard;
 import edu.wpi.first.wpilibj.shuffleboard.ShuffleboardTab;
 import edu.wpi.first.wpilibj.DigitalInput;
 import edu.wpi.first.wpilibj.Servo;
 
+import edu.wpi.first.wpilibj2.command.ProfiledPIDSubsystem;
+import edu.wpi.first.wpilibj2.command.ProfiledPIDCommand;
+
+
+
+
 public class ArmContainer implements BaseContainer {
+
   
     // todo: add intake subsystem
     private final CommandXboxController m_driverController =
         new CommandXboxController(OperatorConstants.kDriverControllerPort);
-        //ArmSubsystem arm = new ArmSubsystem(18, "CAN_Network", 23, "rio", 0, 1, 0); 
+        //CAN_Network
+        ArmSubsystem arm = new ArmSubsystem(ArmSubsystem.Arm.ARM_ID, "CAN_Network", ArmSubsystem.Arm.PIZZABOX_ID, "rio", 
+                        ArmSubsystem.Arm.ENCODER_DIO_SLOT, ArmSubsystem.Arm.SERVO_PWN_SLOT); 
 
     public ArmContainer() {
         configureBindings();
 
     }
 
-
     private void configureBindings() {
-    //    m_driverController.a().onTrue(new SwingForward(arm, 270, 10, 10, .25));
-    //    m_driverController.b().onTrue(new SwingBack(arm, 10, 10, .25));
-        // m_driverController.x().onTrue(new SwingServo(arm));
+        //m_driverController.x().onTrue(new SwingForwardServo(arm));
+        m_driverController.start().onTrue(Commands.runOnce(()-> {
+            if(arm.isEnabled())
+            {
+                arm.disable();
+            }
+            else
+            {
+                arm.enable();
+            }
+        }, arm));
+
+        m_driverController.a().onTrue(Commands.runOnce(() -> {
```

Comment:
These commands and the ones below use the arm constants, which we have yet to set.